nixos/doc/manual/installation/installing.chapter.md

```diff
@@ -1,30 +1,147 @@
 # Installing NixOS {#sec-installation}
 
-## Booting the system {#sec-installation-booting}
+## Booting from the install medium {#sec-installation-booting}
+
+To begin the installation, you have to boot your computer from the install drive.
+
+1.   Plug in the install drive. Then turn on or restart your computer.
+
+2.   Press <kbd>F12</kbd> (or the appropriate key) to open the boot menu.
+     Select the USB flash drive (the option usually contains the word "USB").
+     If you choose the incorrect drive, your computer will likely continue to
+     boot as normal. In that case restart your compute and pick a different drive.
+
+     ::: {.note}
+     The key to open the boot menu is different across computer brands and even
+     models. It is usually <kbd>F12</kbd>, but sometimes <kbd>Esc</kbd> or
+     another function key. If you are unsure, look at the POST screen that is
+     shown first when the computer is started. You can also search online for
+     your computers brand, model followed by "boot from usb".
+     The computer might not even have that feature, so you have to go into the
+     BIOS/UEFI settings to change the boot order. Again, search online for
+     details about your specific computer model.
+
+     For Apple computers with Intel processor press and hold the <kbd>⌥</kbd>
+     (Option or Alt) key until you see the boot menu. On Apple silicon press
+     and hold the power button.
+     :::
+
+     ::: {.note}
+     If your computer support both BIOS and UEFI boot, choose the UEFI option.
+     :::
+
+     ::: {.note}
+     If you use a CD for the installation, the computer will probably boot from
+     it automatically. If not, choose the option containing the word "CD" from
+     the boot menu.
+     :::
+
+3.   Shortly after selecting the appropriate boot drive, you should be
+     presented with a menu with different installer options. Leave the default
+     and wait (or press <kbd>Enter</kbd> to speed up).
+
+4.   The graphical images will start their corresponding desktop environment
+     and the graphical installer, which can take some time. The minimal images
```

Comment:
```suggestion
     and the graphical installer. The minimal images
```
It shouldn't really take long and be done in 30 seconds at most.

Reply:
When letting test people do the install, they where often impatient and thought they have to do something in the black screen before the desktop is shown... so i think this information is essential for people to patiently wait for the desktop to show up! (even if they might not read the manual)

Reply:
Yeah, kde sometimes take a bit to show something. Especially when you are not used to boot from USB.